sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/TagService.java

```diff
@@ -52,7 +51,7 @@ public Tag createTag(Tag tag) {
     @Transactional(rollbackFor = Exception.class)
     public Set<Tag> createTags(Set<Tag> tags) {
         Set<Tag> result = new HashSet<>();
-        if (CollectionUtils.isNotEmpty(tags)) {
+        if (!(tags == null || tags.isEmpty())) {
```

Comment:
maybe make it sense to have condition more readable like `tags != null && !tags.isEmpty()`

Reply:
good point